src/t8.h

```diff
@@ -98,7 +98,8 @@ typedef int32_t t8_locidx_t;
 #define T8_LOCIDX_MAX INT32_MAX
 /** Comparison function for t8_locidx_t */
 #define t8_compare_locidx(v, w) sc_int32_compare (v, w)
-
+/** A type for holding process ids. */
+typedef int t8_procidx_t;
```

Comment:
Please double check: 
I think for uniform_bounds etc. we use a t8_gloidx_t for process ids.